src/TransCreator/numericInputValidator.js

```diff
@@ -0,0 +1,16 @@
+export default (val) => {
```

Comment:
this is not a validator bit a formatter or decorator or something

Reply:
fixed